templates/core/index.html

```diff
@@ -1,817 +1,753 @@
+        </div>
+        <div class="col">
+          <p>
+            <a href="/engage/intro-to-ubuntu-core22-webinar#:~:text=With%204%20releases%20behind%2C%20Ubuntu,edge%20devices%20makers%20must%20face.">Introduction to Ubuntu Core 22</a>
+          </p>
+        </div>
+      </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip" style="padding-bottom: 150px;">
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h3 class="p-heading--4">One platform from cloud to device</h3>
-      <p class="p-heading--5">Bring apps from Ubuntu Server to Ubuntu Core</p>
-      <p>Snaps run on Ubuntu Server, Desktop and Core. One platform, one process. Your developer workstation, your build farm, your cloud and servers all use snaps. Your appliances can too, with extra security.</p>
-      <p><a class="p-button" href="https://snapcraft.io/">Build your IoT application with Snapcraft</a></p>
-    </div>
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
-      {{
-      image(
-      url="https://assets.ubuntu.com/v1/3089fff7-one+platform.svg",
-      alt="",
-      height="230",
-      width="350",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-
-  <div class="row u-vertically-center">
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
-      {{
-      image(
-      url="https://assets.ubuntu.com/v1/6ec5c97f-embedded-linux_AW.svg",
-      alt="",
-      height="200",
-      width="332",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
-    <div class="col-6">
-      <h3 class="p-heading--4">Embedded Linux is easy on Ubuntu</h3>
-      <p class="p-heading--5">We handle the board, you handle the apps</p>
-      <p>No bad BSPs. No maintenance nightmares. No integration delays. Just develop on Ubuntu. Embedded, but not as you knew it.</p>
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h3 class="p-heading--4">Deploy to devices as fast as the cloud</h3>
-      <p class="p-heading--5">Continuous deployment on devices</p>
-      <p>Put your devices on rails for rapid iteration with continuous deployment pipelines, beta testing and canary updates. Featuring Travis integration and a multi-arch build service.</p>
+<section class="p-strip is-deep u-no-padding--top">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Ready to innovate?</h2>
+    </div>
+    <div class="col">
+      <div class="-section--shallow">
+        <p>Whether you are a startup bringing your concept to market or already have large fleets of devices deployed in the field, we have the expertise and infrastructure to launch and support you.</p>
+      </div>
+      <hr class="is-muted p-rule" />
```

Comment:
```suggestion
      <hr class="is-muted" />
```
No `p-rule` when hr is followed by a button or they'll touch like so 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/97265bf5-955f-429e-8166-18d9204f6671)